New global configs (#3180)
fishtown-analytics/docs.getdbt.com: website/docs/reference/global-configs.md

$ dbt --log-format json run
{"code": "A001", "data": {"v": "=1.0.0"}, "invocation_id": "1193e449-4b7a-4eb1-8e8e-047a8b3b7973", "level": "info", "log_version": 1, "msg": "Running with dbt=1.0.0", "node_info": {}, "pid": 35098, "thread_name": "MainThread", "ts": "2021-12-03T10:46:59.928217Z", "type": "log_line"}
```
<VersionBlock firstVersion="1.5">

> To set the `LOG_FORMAT` type output for the file without impacting the console log format, use the `-file` subcommand.

Reviewer: ```suggestion
To set the `LOG_FORMAT_FILE` type output for the file without impacting the console log format, use the `log-format-file` flag.
```

```text
$ dbt --log-format-file json run

> 

Reviewer: Removing extra newline:
```suggestion
```

</File>

<VersionBlock firstVersion="1.5">

### Log Level

> The `LOG_LEVEL` config sets the minimum severity of events captured in the console and file logs. This is a more flexible alternative to the `-debug` log option. The available options for the log levels are `debug`, `info`, `warn`, `error`, or `none`.

Reviewer: ```suggestion
The `LOG_LEVEL` config sets the minimum severity of events captured in the console and file logs. This is a more flexible alternative to the `--debug` flag. The available options for the log levels are `debug`, `info`, `warn`, `error`, or `none`.
```

Setting the `--log-level` will configure console and file logs. 

```text

> $ dbt  --log-level debug run

Reviewer: Removing extra space:
```suggestion
$ dbt --log-level debug run
```

```

> To set the file log level as a different value than the console, use the `-file` subcommand. 

Reviewer: ```suggestion
To set the file log level as a different value than the console, use the `--log-level-file` flag. 
```

```text

> $ dbt  --log-level-file error run

Reviewer: Removing extra space:

```suggestion
$ dbt --log-level-file error run
```

$ dbt --use-colors run
$ dbt --no-use-colors run
```
<VersionBlock firstVersion="1.5">

> You can set the color preferences for the file logs only using the `-file` subcommand.

Reviewer: ```suggestion
You can set the color preferences for the file logs only using the `--use-colors-file / --no-use-colors-file` flags.
```
